locales/zhCN.lua

```diff
@@ -57,231 +57,231 @@ local L = app.L;
-	--TODO: L.TSM_WARNING_1 = "Running this command can potentially destroy your existing TSM settings by reassigning items to the ";
-	--TODO: L.TSM_WARNING_2 = " preset.\n\nWe recommend that you use a different profile when using this feature.\n\nDo you want to proceed anyways?";
-	--TODO: L.PRESET_UPDATE_SUCCESS = "Updated the preset successfully.";
-	--TODO: L.SHOPPING_OP_MISSING_1 = "The preset is missing a 'Shopping' Operation assignment.";
-	--TODO: L.SHOPPING_OP_MISSING_2 = "Type '/tsm operations' to create or assign one.";
-	--TODO: L.AUCTIONATOR_GROUPS = "Group-based searches are only supported using Auctionator.";
-	--TODO: L.TSM4_ERROR = "TSM4 is not compatible with ATT yet. If you know how to create Presets like we used to do in TSM3, please whisper Crieve on Discord!";
-	--TODO: L.QUEST_MAY_BE_REMOVED = "Failed to acquire information. This quest may have been removed from the game. ";
+	L.TITLES_DESC = "头衔会在你的战网中被追踪, 但是, 你的角色必须符合某些头衔的条件才能在该角色上使用.";
+	L.SHARED_APPEARANCES_LABEL = "共享外观";
+	L.SHARED_APPEARANCES_LABEL_DESC = "此列表中的物品是以下物品的共享外观. 在唯一外观模式下, 此列表可帮助了解特定物品被标记为已收集的原因.";
+	L.UNIQUE_APPEARANCE_LABEL = "独特外观";
+	L.UNIQUE_APPEARANCE_LABEL_DESC = "此列表中的物品是独特外观. 你必须专门收集这个物品以获得外观.";
+	L.UPON_COMPLETION = "前置条件";
+	L.UPON_COMPLETION_DESC = "以上任务需要完成后才能够完成以下的事情.";
+	L.QUEST_CHAIN_REQ = "任务链要求";
+	L.QUEST_CHAIN_REQ_DESC = "在能够完成最终任务之前需要完成以下任务.\n\n|cffff6512注意: 战网范围内的任务跟踪将导致该窗口的行为不准确!|r";
+	L.AH_SEARCH_NO_ITEMS_FOUND = "搜索中找不到缓存的物品. 展开组并查看物品缓存名称, 然后重试. 使用此搜索只能找到BOE物品.";
+	L.AH_SEARCH_BOE_ONLY = "使用此搜索只能找到BOE物品.";
+	L.TSM_WARNING_1 = "运行此命令可能会通过将物品重新分配到 ";
+	L.TSM_WARNING_2 = " 预设来破坏您现有的TSM设置.\n\n我们建议你在使用此功能时使用不同的配置文件.\n\n你还想继续吗?";
+	L.PRESET_UPDATE_SUCCESS = "更新预设成功.";
+	L.SHOPPING_OP_MISSING_1 = "预设缺少'Shopping'操作分配.";
+	L.SHOPPING_OP_MISSING_2 = "键入'/tsm operations'创建或者分配一个.";
+	L.AUCTIONATOR_GROUPS = "基于群组的搜索只支持使用Auctionator.";
+	L.TSM4_ERROR = "TSM4尚未与ATT兼容. 如果你知道如何像我们以前在TSM3中那样创建预设, 请在Discord上告诉我!";
+	L.QUEST_MAY_BE_REMOVED = "获取信息失败. 这个任务可能已从游戏中移除. ";
 	--TODO: L.MINUMUM_STANDING = "Requires a minimum standing of";
 	--TODO: L._WITH_ = " with ";
 	--TODO: L.MAXIMUM_STANDING = "Requires a standing lower than";
 	--TODO: L.MIN_MAX_STANDING = "Requires a standing between";
 	--TODO: L.AND_ = "And ";
 	--TODO: L._AND = " and";
-	--TODO: L._OTHER_SOURCES = " other sources";
-	--TODO: L.DURING_WQ_ONLY = "This can be completed when the world quest is active.";
-	--TODO: L.COMPLETED_DAILY = "This can be completed daily.";
-	--TODO: L.COMPLETED_WEEKLY = "This can be completed weekly.";
-	--TODO: L.COMPLETED_MONTHLY = "This can be completed monthly.";
-	--TODO: L.COMPLETED_YEARLY = "This can be completed yearly.";
+	L.DURING_WQ_ONLY = "这个可以在世界任务激活时完成.";
+	L.COMPLETED_DAILY = "这个可以每天完成.";
+	L.COMPLETED_WEEKLY = "这个可以每周完成.";
+	L.COMPLETED_MONTHLY = "这个可以每月完成.";
+	L.COMPLETED_YEARLY = "这个可以每年完成.";
+	L.COMPLETED_MULTIPLE = "这个可以重复多次.";
 	--TODO: L.COMPLETED_MULTIPLE = "This can be repeated multiple times.";
```

Comment:
```suggestion
	--TODO: L._OTHER_SOURCES = " other sources";
	L.DURING_WQ_ONLY = "这个可以在世界任务激活时完成.";
	L.COMPLETED_DAILY = "这个可以每天完成.";
	L.COMPLETED_WEEKLY = "这个可以每周完成.";
	L.COMPLETED_MONTHLY = "这个可以每月完成.";
	L.COMPLETED_YEARLY = "这个可以每年完成.";
	L.COMPLETED_MULTIPLE = "这个可以重复多次.";
```

You have accidentally deleted line `--TODO: L._OTHER_SOURCES = " other sources";` and left `--TODO: L.COMPLETED_MULTIPLE` untouched even though you have translated it above, that's why the number of lines matches.